### DISCUSSION TITLE
Nuxt3 Fetch Error when Posting Form Data with Images

### DISCUSSION THREAD

**Discussion by chesterher11206** [nuxt/nuxt#22650] [Questions]
**Environment**

- Node version: `18.5.0`
- Nuxt version: `3.3.2`

I am using nuxt3 and I have set up an API under the `server` folder to upload images. The API code is like the following:
```typescript
export default defineEventHandler(async (event) => {
  const body = (await readMultipartFormData(event)) || []
  const formData = new FormData()
  for (const item of body) {
    formData.append(
      item.name,
      new Blob([item.data], { type: item.type }),
      item.filename
    )

  await $fetch(
    'external/api/to/upload/image',
    {
      method: 'post',
      body: formData,
    }
  )
})
```

It works fine when I run `nuxt dev` on my Macbook. However, when I deploy it on k8s cluster and run `nuxt build` and `node .output/server/index.mjs`, it still works fine when I upload images where each image size is under around 50 kb, but if the size of a single image is bigger than around 50kb, it will come out the following error:
```bash
FetchError: fetch failed
    at async file:///app/.output/server/chunks/batch.post.mjs:49:9
    at async Object.handler (file:///app/.output/server/node_modules/h3/dist/index.mjs:1247:19)
    at async Server.toNodeHandle (file:///app/.output/server/node_modules/h3/dist/index.mjs:1322:7)
```

Please help me to figure out the root cause (whether it is the frontend problem or the external API problem) and resolve it, thanks!

**Comment by TheAlexLichter** [comment]
A first step would be to try and check the error

```ts
try {
 await $fetch(
    'external/api/to/upload/image',
    {
      method: 'post',
      body: formData,
    }
  )
} catch(e) {
  console.error(e.data)
}
```
